libc/vga/vga-font-default.c

```diff
@@ -0,0 +1,1441 @@
+/* ****** AUTOMATICALLY GENERATED ******
+ * by bdf2c-in-awk  https://gitlab.com/tkchia/bdf2c-in-awk
+ *
+ * Command line arguments:
+ * NONASCII=0 SPARSE=1 COSMO=1 X=_vga_ C=... .../8x13B.bdf
+ * 
+ * Font information:
+ * COMMENT "$ucs-fonts: 8x13B.bdf,v 1.28 2002-11-15 18:23:23+00 mgk25 Rel $"
+ * COMMENT "Send bug reports to Markus Kuhn <http://www.cl.cam.ac.uk/~mgk25/>"
+ * COPYRIGHT "Public domain font.  Share and enjoy."
+ * 
+ * Extra comments:
+ * + xorg.freedesktop.org/releases/individual/font/font-misc-misc-1.1.1.tar.bz2
+ */
+const uint8_t _vga_font_default_direct[95][13] = {
+  [0] = { /* 32 */
+    0x00, /* ........ */
+    0x00, /* ........ */
+    0x00, /* ........ */
+    0x00, /* ........ */
+    0x00, /* ........ */
+    0x00, /* ........ */
+    0x00, /* ........ */
+    0x00, /* ........ */
+    0x00, /* ........ */
+    0x00, /* ........ */
+    0x00, /* ........ */
+    0x00, /* ........ */
+    0x00, /* ........ */
+  },
+  [1] = { /* 33 */
+    0x00, /* ........ */
+    0x18, /* ...##... */
```

Comment:
I guess this has a nice retro aesthetic that'll help us make progress until we can have truetype.